tests/test_config_gsobject.py

```diff
@@ -78,6 +79,11 @@ def test_gaussian():
     gal5b = gal5b.rotate(-34 * galsim.degrees).lens(-0.15, 0.2, 0.93)
     gsobject_compare(gal5a, gal5b)
 
+    # DeltaFunction is functionally equivalent to an extremely narrow Gaussian.
```

Comment:
This is true, but shouldn't we also compare with an actual DeltaFunction in addition to comparing with an extremely narrow Gaussian?